edml-1.0.0.json

```
@@ -0,0 +1,215 @@
+            },
+            "description": {
+              "$ref": "#/definitions/description"
+            }
+          },
+          "required": [
+            "mapping"
+          ],
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    },
+    "description": {
+      "type": "string",
+      "title": "Optional description",
+      "description": "Using this property you can provide documentation."
+    },
+    "required": {
+      "type": "boolean",
+      "default": false,
+      "description": "If this property is set to true, queries will abort if the given mapping can't be found a single document. Else, the query will continue and set this cell to a default value. By default this is property is set to false."
+    },
+    "key": {
+      "type": "string",
+      "enum": [
+        "local",
+        "global"
+      ],
+      "description": "Setting this property marks a column as key.\n\n Key columns are used as foreign keys from ToTableMappings for nested lists. \n\nA key can either be local or global. A global key must be unique over all rows of the table. A local key must only be unique in the scope of the nested list."
+    }
+  },
+  "type": "object",
+  "title": "The Root Schema - Represents the mapping from a DynamoDB table to an Exasol Virtual Schema table",
+  "description": "The structure of this mapping definition follows the structure of the data in the DynamoDB. This object represents a DynamoDB table. Its properties are modeled in the fields property of this object. Only properties of the DynamoDB table that shall be mapped are modeled. \n\n Each of this objects must be stored in a single .json file an get uploaded to an directory in bucketfs.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "source",
+    "destinationTable",
+    "mapping"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Reference to the mapping language schema. Setting this is required so that IDEs can help you when writing your definition.",
+      "enum": [
+        "https://schemas.exasol.com/edml-1.0.0.json",
+        "../../main/resources/schemas/edml-1.0.0.json"
+      ]
+    },
+    "source": {
+      "type": "string",
+      "title": "DynamoDB table name",
+      "description": "The identifier of the DynamoDB table."
+    },
+    "destinationTable": {
+      "$ref": "#/definitions/destinationTable"
+    },
+    "description": {
+      "$ref": "#/definitions/description"
+    },
+    "mapping": {
+      "$ref": "#/definitions/mappingDefinition",
+      "description": "Defines the mapping for the columns of the table. Typically fields is set here to navigate the DynamoDB tables properties. But also toJson mapping is possible here. In that case the whole document is mapped to a JSON string."
```

```suggestion
      "description": "Defines the mapping for the columns of the table. Typically 'fields' is set here to navigate the DynamoDB tables properties. But also 'toJson' mapping is possible here. In that case the whole document is mapped to a JSON string."
```

Generally look trough the descriptions and set tag names in single quotes to make the text more readable.

---

```suggestion
      "description": "Defines the mapping for the columns of the table. Typically 'fields' is set here to navigate the DynamoDB tables properties. But also 'toJsonMapping' mapping is possible here. In that case the whole document is mapped to a JSON string."
```